core/controllers/collection_editor.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+
+# Copyright 2015 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for the collections editor."""
+
+from core.controllers import base
+from core.domain import collection_services
+from core.domain import config_domain
+from core.domain import rights_manager
+from core.platform import models
+import utils
+
+current_user_services = models.Registry.import_current_user_services()
+
+
+def _require_valid_version(version_from_payload, collection_version):
```

Comment:
Any reason to make this module-level and not put it in WritableCollectionDataHandler, which I believe is the only place which uses it?